### DISCUSSION TITLE
Export CSV batch time config [Aug 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3333] [Discord Threads]
> 🙋‍♂️ **niroknox**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Consider increasing config value `NC_EXPORT_MAX_TIMEOUT`   
#### https://docs.nocodb.com/getting-started/installation#environment-variables  
#  
> 🙋‍♂️ **niroknox**  
#### Wow so quick, thank you!   
#### Just to confirm, this would be in the .env and then restart the application, right?   
####    
#### If it's not there, just add it?   
#### So the limit is in time rather than rows... just making sure i got the hang of it..  
#  
> 👨‍🦲 **dstala**  
#### Yes, thats correct. Limit is required to be configured in milliseconds; default is 5000 ms.   
####    
#### If you are using docker, use `-e` option as in   
#### ```   
#### docker run -d --name nocodb-mysql \   
#### -v "$(pwd)"/nocodb:/usr/app/data/ \   
#### -p 8080:8080 \   
#### -e NC_DB="mysql2://host.docker.internal:3306?u=root&p=password&d=d1" \   
#### -e NC_AUTH_JWT_SECRET="569a1821-0a93-45e8-87ab-eb857f20a010" \   
#### nocodb/nocodb:latest   
#### ```  
#  
> 🙋‍♂️ **niroknox**  
#### Not using docker - Just added it tot he file - worked perfectly! thanks!  
#  
> 🙋‍♂️ **evanAndiez**  
#### Good use. It helps me too. This param should be written in nocodb docs 🤓  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1008395386160943154)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
